Chapter7/para_macro.erl

```diff
@@ -0,0 +1,12 @@
+-module(para_macro).
+-export([test/0]).
+
+% Exercise 7-6: Parameterized Macros
+
+-ifdef(show).
+	-define(SHOW_EVAL(Expression), io:format("~p = ~p~n",[??Expression,Expression])).
```

Comment:
This macro doesn't return result of expression. Can you please fix it?
Keep in mind that:
> You should ensure that the expression is evaluated only once whichever case holds.